build.gradle

```diff
@@ -486,9 +486,8 @@ jacoco {
 
 sonarqube {
```

Comment:
@smathangi is it same to revert these changes or it's something you want to keep?

Reply:
ah ok ignore this, that's the doc restore endpoint so we should remove it 👍